keyboards/handwired/drevogramr_qmk/drevogramr_qmk.c

```diff
@@ -0,0 +1,178 @@
+/* Copyright 2021 wwwMADwww <4d4144h@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "quantum.h"
+#include "led.h"
+
+/*==========================================================================
+============================================================================
+==========================================================================*/
+
+int matrixArraySize = MATRIX_ROWS * sizeof(matrix_row_t);
```

Comment:
Generally, all the matrix code is placed in a `matrix.c` file to clarify its purpose.  

It's not required, but it is recommended.

Reply:
`drevogramr_qmk.c` renamed to `matrix.c`